<html>

<head>
	<title>Chloranthales</title>
	<style>
A {text-decoration : none;} 
	.ill {color: #1C960A; font-weight : bold}
</style>
	<SCRIPT LANGUAGE="JavaScript"> 
<!-- 
var win; 
function showImage(img,w,h) { 
   var url = img; 
   windowName = "imagewin"; 
      params  = ""; 
         params += "toolbar=0,"; 
         params += "location=0,"; 
         params += "directories=0,"; 
        params += "status=0,"; 
         params += "menubar=0,"; 
         params += "scrollbars=1,"; 
         params += "resizable=0,"; 
         params += "width="+ (w + 10) +","; 
         params += "height="+ (h + 10); 

   if(!win) { 
      win = window.open(url, windowName, params); 
         win.focus(); } 
         else { 
          win.close(); 
          win = window.open(url, windowName, params); 
         win.focus(); 
         } 
   }
//--> 
</SCRIPT> 
</head>

<body bgcolor="#FFFFFF"><font face="verdana, georgia, arial, serif" size="2">
<body text="Black" link="Blue" vlink="Blue" alink="Blue">


<p><b><font face="verdana, georgia, arial, serif" size="3.5"><a name=Embryophytes>EMBRYOPSIDA</a></font></b> Pirani &amp; Prado</p>

<p>Gametophyte dominant, independent, multicellular, not motile, initially &plusmn;globular; showing gravitropism; <b class="apo">acquisition of phenylalanine lysase</b> [PAL], <b class="apo">microbial terpene synthase-like genes +</b>, triterpenoids produced by CYP716 enzymes, phenylpropanoid metabolism  [lignans +,  flavonoids + (absorbtion of UV radiation)], <b class="apo">xyloglucans in primary cell wall</b>,  side chains charged; <b class="apo">plant poikilohydrous</b> [protoplasm dessication tolerant], <b class="apo">ectohydrous</b> [free water outside plant physiologically important];  thalloid, <b class="apo">leafy</b>, <b class="apo">with single-celled apical meristem</b>, tissues little differentiated, <b class="apo">rhizoids +, unicellular</b>;  chloroplasts several per cell, <b class="apo">pyrenoids 0</b>; <b class="apo">glycolate metabolism in leaf peroxisomes</b> [glyoxysomes]; <b class="apo">centrioles/centrosomes in vegetative cells 0</b>, microtubules with &gamma;-tubulin along their lengths [?here],  interphase microtubules form hoop-like system; <b class="apo">metaphase spindle anastral</b>, predictive <b class="apo">preprophase band +</b> [with  microtubules and F-actin; where new cell wall will form], <b class="apo">phragmoplast +</b> [cell wall deposition centrifugal, from around the anaphase spindle], plasmodesmata +; <b class="apo">antheridia and archegonia jacketed</b>, surficial;  blepharoplast +, centrioles develop <i>de novo</i>, <b class="apo">bicentriole pair coaxial, separate at midpoint, centrioles rotate</b>, associated with basal bodies of cilia, <b class="apo">multilayered structure +</b> [4 layers: L1, L4, tubules; L2, L3, short vertical lamellae] (0),  spline + [tubules from L1 encircling spermatid], basal body 200-250 nm long, associated with amorphous electron-dense material, <b class="apo">microtubules in basal end lacking symmetry</b>, <b class="apo">stellate array of filaments in transition zone extended</b>, <b class="apo">axonemal cap 0</b> [microtubules disorganized at apex of cilium]; male gametes [spermatozoids] with a left-handed coil, cilia 2, lateral; <b class="apo">oogamy</b>; <b class="apo">sporophyte  multicellular</b>, <b class="apo">cuticle +</b>, plane of first cell division transverse [with respect to long axis of archegonium/embryo sac], sporangium and upper part of seta developing from epibasal cell [towards the archegonial neck, exoscopic],  with at least transient apical cell [?level], initially surrounded by and dependent on  gametophyte, <b class="apo">placental transfer cells +</b>, in both sporophyte and gametophyte, wall ingrowths develop early; <b class="apo">suspensor/foot +</b>, cells at foot tip somewhat haustorial; <b class="apo">sporangium +</b>, <b class="apo">single</b>, <b class="apo">terminal</b>, dehiscence longitudinal; meiosis sporic, monoplastidic, MTOC [MTOC = <u>m</u>icro<u>t</u>ubule <u>o</u>rganizing <u>c</u>entre] associated with plastid,  sporocytes 4-lobed, <b class="apo">cytokinesis simultaneous</b>,  preceding nuclear division, <b class="apo">quadripolar microtubule system +</b>; wall development both centripetal and centrifugal, <b class="apo">1000 spores/sporangium</b>, <b class="apo">sporopollenin in the spore wall</b> laid down in association with trilamellar layers [white-line centred lamellae; tripartite lamellae]; nuclear genome size [1C] <1.4 pg, main telomere sequence motif TTTAGGG, <i>LEAFY</i> and <b class="apo"><i>KNOX</i>1</b> and <b class="apo"><i>KNOX</i>2</b>  genes present, ethylene involved in cell elongation; chloroplast genome with close association between <i>trn</i>L<sub>UAA</sub> and <i>trn</i>F<sub>GAA</sub> genes [precursors for starch synthesis], <b class="apo"><i>tuf</i>A gene moved to nucleus</b>; mitochondrial <i>trn</i>S(gcu) and <i>trn</i>N(guu) genes +.</p>


<p style="text-indent: 30px;">Many of the bolded characters in the characterization above are apomorphies of subsets of streptophytes along the lineage leading to the embryophytes, not apomorphies of  crown-group embryophytes <i>per se</i>.</p>

<p style="text-indent: 30px;">All groups below are crown groups, nearly all are extant. Characters mentioned are those of the immediate common ancestor of the group, [] contains explanatory material, () features common in clade, exact status unclear.</p>

<p><font face="verdana, georgia, arial, serif" size="3.5"><b>STOMATOPHYTES</b></font></p>

<p>Abscisic acid, <b class="apo">L- and D-methionine distinguished metabolically</b>; <b class="apo">pro- and metaphase spindles acentric</b>; sporophyte with polar transport of auxins, class 1 KNOX genes expressed in sporangium alone; sporangium wall 4&le; cells across [&equiv; eusporangium], <b class="apo">tapetum +</b>, <b class="apo">secreting sporopollenin</b>, which obscures outer white-line centred lamellae, <b class="apo">columella +</b>, developing from endothecial cells; <b class="apo">stomata +</b>, on sporangium, anomocytic, cell lineage that produces them with symmetric divisions [perigenous]; underlying similarities in the development of conducting tissue and of rhizoids/root hairs; <b class="apo">spores trilete</b>; shoot meristem  patterning gene families expressed; <i>MIKC</i>, <i>MI<sup>*</sup>K<sup>*</sup>C<sup>*</sup></i>  genes, post-transcriptional editing of chloroplast genes; gain of three group II mitochondrial introns, <b class="apo">mitochondrial <i>trn</i>S(gcu) and <i>trn</i>N(guu) genes 0</b>.</p>

<p>[Anthocerophyta + Polysporangiophyta]:  gametophyte leafless; <b class="apo">archegonia embedded/sunken</b> [only neck protruding]; <b class="apo">sporophyte long-lived, chlorophyllous</b>; <b class="apo">cell walls with xylans</b>.</p>


<p><font face="verdana, georgia, arial, serif" size="3.5"><b>POLYSPORANGIOPHYTA&dagger;</b></font></p>

<p><b class="apo">Sporophyte well developed</b>, <b class="apo">branched</b>, branching apical, dichotomous, potentially indeterminate; hydroids +; stomata on stem; <b class="apo">sporangia several</b>, terminal; <b class="apo">spore walls not multilamellate</b> [?here].</p>
<p><font face="verdana, georgia, arial, serif" size="3.5"><b>TRACHEOPHYTA&dagger; </b></font></p>

<p><b class="apo">Vascular tissue +</b> [tracheids, walls with bars of secondary thickening].</p>

<p><font face="verdana, georgia, arial, serif" size="3.5"><b>EXTANT TRACHEOPHYTA / VASCULAR PLANTS</b></font></p>



<p>Sporophyte with photosynthetic red light response, stomata open in response to blue light; <b class="apo">plant homoiohydrous</b> [water content of protoplasm relatively stable]; control of leaf hydration passive;  <b class="apo">plant endohydrous</b> [physiologically important free water inside plant]; (condensed or nonhydrolyzable tannins/proanthocyanidins +); xyloglucans with side chains uncharged [?level],  in secondary walls of vascular and mechanical tissue; <b class="apo">lignins +</b>; <b class="apo">stem apex multicellular</b>, with cytohistochemical zonation, plasmodesmata formation based on cell lineage; <b class="apo">tracheids +</b>, in both protoxylem and metaxylem, G- and S-types;  <b class="apo">sieve cells +</b> [nucleus degenerating]; endodermis +; leaves/sporophylls spirally arranged, blades with mean venation density ca 1.8 mm/mm<sup>2</sup> [to 5 mm/mm<sup>2</sup>], all epidermal cells with chloroplasts; <b class="apo">sporangia adaxial</b>, <b class="apo">columella 0</b>; tapetum glandular; ?position of transfer cells; <b class="apo">MTOCs not associated with plastids</b>, <b class="apo">basal body 350-550 nm long</b>, <b class="apo">stellate array in transition region initially joining microtubule triplets</b>;  <b class="apo">suspensor +</b>, <b class="apo">shoot apex developing away from micropyle/archegonial neck</b> [from hypobasal cell, endoscopic], root lateral with respect to the longitudinal axis of the embryo [plant homorhizic].</p>

<b>[MONILOPHYTA + LIGNOPHYTA]</b> </p>


<p>Sporophyte  endomycorrhizal [with Glomeromycota]; growth &plusmn; monopodial, branching spiral; <b class="apo">roots +</b>, endogenous, positively geotropic, root hairs and root cap +, protoxylem exarch, <b class="apo">lateral roots +</b>, endogenous;  G-type tracheids +, with scalariform-bordered pits; leaves with apical/marginal growth, venation development basipetal, growth determinate; sporangium dehiscence by a single longitudinal slit; <b class="apo">cells polyplastidic</b>, <b class="apo">MTOCs diffuse, perinuclear</b>, migratory; <b class="apo">blepharoplasts +, paired</b>, with electron-dense material, centrioles on periphery, <b class="apo">male gametes multiciliate</b>; <b class="apo">chloroplast long single copy ca 30kb inversion</b> [from <i>psb</i>M to <i>ycf</i>2];  <b class="apo">mitochondrion with loss of 4 genes</b>, <b class="apo">absence of numerous group II introns</b>; <b class="apo">LITTLE ZIPPER proteins</b>.</p>

<p><b><font face="verdana, georgia, arial, serif" size="3.5">LIGNOPHYTA&dagger;</font></b></p>

<p>Sporophyte woody; <b class="apo">stem branching lateral, meristems axillary</b>; lateral root origin from the pericycle; <b class="apo">cork cambium +</b> [producing cork abaxially], <b class="apo">vascular cambium bifacial</b> [producing phloem abaxially and xylem adaxially].</p>

<p><b><font face="verdana, georgia, arial, serif" size="3.5">SEED PLANTS&dagger;</font></b></p>

<p><b class="apo">Plants heterosporous</b>; <b class="apo">megasporangium surrounded by cupule</b> [i.e. = unitegmic ovule, cupule = integument]; pollen lands on ovule; <b class="apo">megaspore germination endosporic</b> [female gametophyte initially retained on the plant].</p>

<p><b><font face="verdana, georgia, arial, serif" size="3.5">EXTANT SEED PLANTS / SPERMATOPHYTA</font></b></p>

<p>Plant evergreen; nicotinic acid metabolised to trigonelline, (cyanogenesis via tyrosine pathway); <b class="apo">microbial terpene synthase-like genes 0</b>; primary cell walls rich in xyloglucans and/or glucomannans, 25-30% pectin [Type I walls]; lignin chains started by monolignol dimerization [resinols common],   particularly with guaiacyl and <i>p</i>-hydroxyphenyl [G + H] units [sinapyl units uncommon, no Ma&uuml;le reaction]; root stele diarch to pentarch, xylem and phloem originating on alternating radii, cork cambium deep seated; <b class="apo">stem apical meristem complex</b> [with quiescent centre, etc.],  plasmodesma density in SAM 1.6-6.2[mean]/&mu;m<sup>2</sup> [interface-specific plasmodesmatal network]; <b class="apo">eustele +</b>, protoxylem endarch, endodermis 0; wood homoxylous, tracheids and rays alone, tracheid/tracheid pits circular, bordered; <b class="apo">mature sieve tube/cell lacking functioning nucleus</b>, sieve tube plastids with starch grains; phloem fibres +; cork cambium superficial; leaf nodes 1:1, <b class="apo">a single trace leaving the vascular sympodium</b>; leaf vascular bundles amphicribral;  <b class="apo">guard cells the only epidermal cells with chloroplasts</b>, <b class="apo">stomatal pore with active opening in response to leaf hydration</b>, <b class="apo">control by abscisic acid</b>, <b class="apo">metabolic regulation of water use efficiency</b>, etc.; axillary buds +,  exogenous; prophylls two, lateral; <b class="apo">leaves with petiole and lamina</b>, development basipetal, lamina simple; sporangia borne on sporophylls; <b class="apo">spores not dormant</b>; microsporophylls aggregated in indeterminate cones/strobili; <b class="apo">grains monosulcate</b>, <b class="apo">aperture in ana- position</b> [distal],  <b class="apo">primexine +</b> [involved in exine pattern formation with deposition of sporopollenin from tapetum there], <b class="apo">exine and intine homogeneous</b>, exine alveolar/honeycomb; ovules with parietal tissue [= crassinucellate], <b class="apo">megaspore tetrad linear</b>, <b class="apo">functional megaspore single</b>, <b class="apo">chalazal</b>, <b class="apo">sporopollenin 0</b>; <b class="apo">gametophyte &plusmn; wholly dependent on sporophyte</b>,  <b class="apo">development initially endosporic</b> [apical cell 0, rhizoids 0, etc.]; male gametophyte with tube developing from distal end of grain,  <b class="apo">male gametes  two</b>, developing after pollination, with cell walls; <b class="apo">female gametophyte initially syncytial</b>, walls then surrounding individual nuclei; <b class="apo">embryo cellular ab initio</b>, suspensor short-minute, <b class="apo">embryonic axis straight</b> [shoot and root at opposite ends; plant allorhizic], cotyledons 2; <b class="apo">embryo &plusmn; dormant</b>; chloroplast  <b class="apo"><i>ycf</i>2 gene in inverted repeat</b>, <b class="apo"><i>trans</i> splicing of five mitochondrial group II introns</b>, <b class="apo"><i>rpl</i>6 gene absent</b>; <b class="apo">whole nuclear genome duplication</b> [&zeta; - zeta - duplication], two copies of <i>LEAFY</i> gene, <i>PHY</i> gene duplications [three - [<i>B</i><i>P</i> [<i>A</i>/<i>N</i> + <i>C</i>/<i>O</i>]] - copies], 5.8S and 5S rDNA in separate clusters.</p>

<p><b><font face="verdana, georgia, arial, serif" size="3.5">ANGIOSPERMAE / MAGNOLIOPHYTA</font></b></p>

<p>Lignans, O-methyl flavonols, dihydroflavonols, <b class="apo">triterpenoid oleanane</b>, apigenin and/or luteolin scattered, [cyanogenesis in ANA grade?], <b class="apo">lignin also with syringyl units common</b> [G + S lignin, positive Ma&uuml;le reaction - syringyl:guaiacyl ratio more than 2-2.5:1], hemicelluloses as xyloglucans; root cap meristem closed (open); pith relatively inconspicuous, lateral roots initiated immediately to the side of [when diarch] or opposite xylem poles; origin of epidermis with no clear pattern [probably from inner layer of root cap], trichoblasts [differentiated root hair-forming cells] 0, <b class="apo">hypodermis suberised and with Casparian strip</b> [= exodermis]; <b class="apo">shoot apex with tunica-corpus construction, tunica 2-layered</b>;  starch grains simple; primary cell wall mostly with pectic polysaccharides, poor in mannans; tracheid:tracheid [end wall] plates with scalariform pitting, wood parenchyma +; <b class="apo">sieve tubes enucleate</b>, sieve plate with pores (0.1-)0.5-10< &micro;m across, <b class="apo">cytoplasm with P-proteins</b>, not occluding pores of plate, <b class="apo">companion cell and sieve tube from same mother cell</b>; ?phloem loading/sugar transport; nodes 1:?; dark reversal P<sub>fr</sub> &#8594 P<sub>r</sub>; protoplasm dessication tolerant [plant poikilohydric]; <b class="apo">stomata brachyparacytic</b> [ends of subsidiary cells level with ends of pore], outer stomatal ledges producing vestibule, <b class="apo">reduction in stomatal conductance with increasing CO<sub>2</sub> concentration</b>; lamina formed from the
primordial leaf apex, margins toothed, development of venation acropetal, <b class="apo">overall growth &plusmn; diffuse</b>, secondary veins pinnate, <b class="apo">fine venation hierarchical-reticulate</b>, (1.7-)4.1(-5.7) mm/mm<sup>2</sup>, vein endings free; <b class="apo">flowers perfect</b>, pedicellate, &plusmn; haplomorphic, protogynous; parts free, numbers variable, development centripetal; <b class="apo">P +</b>, ?insertion, members each with a single trace, outer members not sharply differentiated from the others, not enclosing the floral bud; A many, filament not sharply distinguished from anther, stout, broad, with a single trace, <b class="apo">anther</b> introrse, <b class="apo">tetrasporangiate, sporangia in two groups of two</b> [dithecal], <b class="apo">each theca dehiscing longitudinally by a common slit</b>, <b class="apo">&plusmn; embedded in the filament</b>, walls with at least outer secondary parietal cells dividing, <b class="apo">endothecium +</b>, cells elongated at right angles to long axis of anther; tapetal cells binucleate; microspore mother cells in a block, <b class="apo">microsporogenesis successive</b>, walls developing by centripetal furrowing; pollen subspherical, tectum continuous or microperforate, <b class="apo">ektexine columellate</b>, <b class="apo">endexine lamellate only in the apertural regions</b>, thin, compact, intine in apertural areas thick, <b class="apo">pollenkitt +</b>; nectary 0; <b class="apo">carpels present</b>, superior, free, several, <b class="apo">ascidiate</b> [postgenital occlusion by secretion], stylulus at most short [shorter than ovary], hollow, cavity not lined by distinct epidermal layer, <b class="apo">stigma &plusmn; decurrent</b>, carinal, dry; <b class="apo">suprastylar extragynoecial compitum +</b>; <b class="apo">ovules few [?1]/carpel</b>, marginal, <b class="apo">anatropous</b>, <b class="apo">bitegmic</b>, <b class="apo">micropyle endostomal</b>, <b class="apo">outer integument 2-3 cells across</b>, often largely subdermal in origin, <b class="apo">inner integument 2-3 cells across</b>, often dermal in origin,  parietal tissue 1-3 cells across,  nucellar cap?; <b class="apo">megasporocyte single</b>, <b class="apo"> hypodermal</b>, functional megaspore  lacking cuticle;  <b class="apo">female gametophyte lacking chlorophyll</b>, not  photosynthesising, <b class="apo">four-celled</b> [one module, nucleus of egg cell sister to one of the polar nuclei]; ovule not increasing in size between pollination and fertilization; <b class="apo">pollen grains land on stigma</b>, <b class="apo">bicellular at dispersal</b>, <b class="apo">mature male gametophyte tricellular</b>, germinating in less than 3 hours, pollen tube elongated, unbranched, <b class="apo">growing between cells</b>, <b class="apo">growth rate (20-)80-20,000 &micro;m/hour</b>, apex of pectins, <b class="apo">wall with callose</b>, <b class="apo">lumen with callose plugs</b>, <b class="apo">penetration of ovules via micropyle</b> [porogamous], <b class="apo">whole process takes ca 18 hours</b>, distance to first ovule 1.1-2.1 mm; <b class="apo">male gametes lacking cell walls</b>, <b class="apo">ciliae 0</b>, <b class="apo">siphonogamy</b>; <b class="apo">double fertilization +</b>, ovules aborting unless fertilized; P deciduous in fruit; mature seed much larger than fertilized ovule, small [], dry [no sarcotesta], exotestal; <b class="apo">endosperm +</b>, cellular, development heteropolar [first division oblique, micropylar end initially with a single large cell, divisions uniseriate, chalazal cell smaller, divisions in several planes], copious, oily and/or proteinaceous, embryo short [&lt;&frac14; length of seed]; plastid and mitochondrial transmission maternal; <i>Arabidopsis</i>-type telomeres [(TTTAGGG)<sub>n</sub>]; nuclear genome size [1C] <1.4 pg [mean 1C = 18.1 pg, 1 pg = 10<sup>9</sup> base pairs], <b class="apo">whole nuclear genome duplication</b> [&epsilon;/epsilon event]; <b class="apo"><i>ndh</i>B gene 21 codons enlarged at the 5' end</b>, single copy of <i>LEAFY</i> and <i>RPB</i>2 gene, <i>knox</i> genes extensively duplicated [A1-A4], <i>AP1/FUL</i> gene, <b class="apo">palaeo <i>AP3</i> and <i>PI</i> genes</b> [paralogous B-class genes] +, <b class="apo">with "DEAER" motif</b>, <b class="apo"><i>SEP</i>3/<i>LOFSEP</i> and three copies of the PHY gene</b>, [<i>PHYB</i> [<i>PHYA</i> + <i>PHYC</i>]]; chloroplast <b class="apo"><i>chl</i>B, -L, -N, <i>trn</i>P-GGG genes 0</b>.</p>




<p>[NYMPHAEALES [AUSTROBAILEYALES [[CHLORANTHALES + MAGNOLIIDS]
 [MONOCOTS [CERATOPHYLLALES + EUDICOTS]]]]]: wood fibres +; axial parenchyma diffuse or diffuse-in-aggregates; <b class="apo">pollen monosulcate</b> [anasulcate], tectum reticulate-perforate [here?]; ?genome duplication; "DEAER" motif in <i>AP3</i> and <i>PI</i> genes lost, gaps in these genes.</p>

<p>[AUSTROBAILEYALES [[CHLORANTHALES + MAGNOLIIDS]
 [MONOCOTS [CERATOPHYLLALES + EUDICOTS]]]]:  phloem loading passive, via symplast, plasmodesmata numerous; <b class="apo">vessel elements with scalariform perforation plates</b> in primary xylem; <b class="apo">essential oils in specialized cells</b> [lamina and P &plusmn; pellucid-punctate]; <b class="apo">tension wood +</b> [reaction wood: with gelatinous fibres, G-fibres, on adaxial side of branch/stem junction]; <b class="apo">tectum reticulate</b>; anther wall with outer secondary parietal cell layer dividing; nucellar cap + [character lost where in eudicots?]; 12BP [4 amino acids] deletion in <i>P1</i> gene.</p>


<p><a name="ChltoRest">[[CHLORANTHALES + MAGNOLIIDS] [MONOCOTS [CERATOPHYLLALES + EUDICOTS]]] / MESANGIOSPERMAE: <b class="apo">benzylisoquinoline alkaloids +</b>; sesquiterpene synthase subfamily a [TPS-a] [?level], polyacetate derived anthraquinones + [?level]; outer epidermal walls of root elongation zone with cellulose fibrils oriented transverse to root axis; P more or less whorled, 3-merous [?here]; <b class="apo">pollen tube growth intra-gynoecial</b>; extragynoecial compitum 0; carpels plicate [?here]; <b class="apo">embryo sac bipolar, 8 nucleate</b>, antipodal cells persisting; <b class="apo">endosperm triploid</b>.</a>&nbsp;&nbsp;<a href="treeapweb2map.html" target="home">Back to Main Tree</a></p>


<p style="text-indent: 30px;"><b>Age.</b> Clarke et al. (2011: see other estimates) suggested an age of (179-)152(-133) m.y. for this clade, N. Zhang et al. (2012) an age  of (163-)145(-133) m.y., Xue et al. (2012) an age of ca 146.4 m.y., Naumann et al. (2013) an age of about 148.5 m.y., and Magall&oacute;n et al. (2013) an age of around (180.7-)158.7-151.6(-137.7) m. years. Some other estimates are older, ranging from ca 176 m.y. (Foster et al. 2016a, q.v. for details) to (200-)174(-153) m.y. (with eudicot calibration) or (210-)184(-160)  m.y. (without: Smith et al. 2010). Looking at the pattern of duplication of <i>SEPALLATA</i> genes, Yockteng et al. (2013) dated this node to around 187-137.4 m.y.a. and Magall&oacute;n et al. (2015) to around 135.9 m.y.a.; (216, 197-)191, 154(-141) and ca 217 m.y. are the somewhat older spread of ages on Zheng et al. (2014) and Z. Wu et al. (2014) respectively, and  ca 219 m.y. in Tank et al. (2015: Table S1). Bell et al. (2010: Chloranthaceae sister to [Magnoliidae + everything else], but not monocots) suggested ages for this node of (152-)140(-128) or (135-)127(-119) m.y. depending on the method used. </p>

<p style="text-indent: 30px;">The age of this branching point is estimated at (190-)167(-47) m.y. (95% HPD, Smith et al. 2010, Table S3), while a fossil-based estimate is ca 100 m.y. (Crepet et al. 2004). [Check]</p>




<p style="text-indent: 30px;"><b>Evolution:</b> <u>Divergence &amp; Distribution.</u> Tank et al. (2015) suggest that an increase in net diversification at this node might be linked to the &epsilon;/epsilon whole genome duplication that characterises all <a href="amborellalesweb2.htm#Magnoliophyta" target="home">angiosperms</a>; see also P. Soltis and Soltis (2016) for floral evolution. If [Chloranthales + Ceratophyllales] - or just Chloranthaceae - are sister to all other mesangiosperms (see below), this increase will have to be placed at the node above their place of departure.</p>

<p style="text-indent: 30px;">For the distribution of isoquinoline alkaloids, alternatively known as 1-benzyltetrahydroisoquinoline or 1-btiq alkaloids, see Waterman (1999, 2007). How to optimise them on the tree is unclear. They are known from Chloranthaceae, the magnoliids, and eudicots, so if there is a clade [[Chloranthaceae + magnoliids] [monocots [Ceratophyllaceae + eudicots]]], as is recognized above, or some variation of this, they may be best optimised here. </p>


<p style="text-indent: 30px;"><u>Ecology &amp; Physiology.</u>  Primary xylem with simple perforation plates in both protoxylem and metaxylem in the leaves has evolved several times within this clade (Feild &amp; Brodribb 2013); ceteris paribus, such vessels conduct water well. This then allowed the miniaturization of the foliar vein reticulum through the development of veins that are narrow yet still conduct water adequately, in turn allowing dense leaf venation and all that this entails in terms of high rates of photosynthesis and evaporation, etc.; see also <a href="amborellalesweb2.htm#VenDen" target="home">elsewhere</a>. </p>

<p style="text-indent: 30px;"><u>Genes &amp; Genomes.</u> Evans and Rees (1971) discuss variation in the length of the mitotic cycle, with that in eudicots being ca 4 hours longer than that in monocots (interphase, G1, is involved, 16 species sampled). </p>


<p style="text-indent: 30px;"><b>Chemistry, Morphology, etc.</b>  The betalains of core Caryophyllales have biosynthetic similarities with benzylisoquinoline alkaloids. For the sesquiterpene synthase subfamily a, see the <i>Amborella</i> Genome Project (2013); not in Amborellaceae. For the orientation  of cellulose fibrils in the outer epidermal walls of root elongation zone, see Kerstens and Verbelen (2002); I do not know what happens in the ANITA grade and in gymnosperms, and magnoliids and eudicots are very poorly sampled. This is perhaps the best place to put triploid endosperm on the tree; the other would be as a synapomorphy for all angiosperms, but in that case it would subsequently be lost twice, or lost once and then regained.</p>


<p style="text-indent: 30px;">I largely follow Ronse De Craene et al. (2003) for the insertion of floral organs. To add where?: A whorled.</p>

<p style="text-indent: 30px;"><b>Phylogeny.</b> Relationships between the major clades mmediately above the basal pectination, the ANITA grade (Amborellales, Nymphaeales and Austrobaileyales), remain unclear (see also Zeng et al. 2014; Wickett et al. 2014 for a summary). 
Pending further studies, apomorphies for a topology [[Chloranthales + magnoliids] [monocots [Ceratophyllales + eudicots]]] are suggested, but the main tree shows a polytomy, the relationships of these five groups being unresolved (see also P. Soltis et al. 1999; D. Soltis et al. 2005b; c.f. A.P.G. IV 2016, and in particular A.P.G. III 2006). A variety of other topologies have also been described.  For further details of the clades involved, see <a href="ranunculalesweb.htm#Eudicots" target="home">eudicots</a>,  <a href="magnolialesweb.htm#Maglaucanpip" target="home">magnoliids</a>, <a href="ceratophyllalesweb.htm#Ceratophyllales" target="home">Ceratophyllales</a>, <a href="chloranthalesweb.htm#Chloranthales" target="home">Chloranthales</a> and <a href="acoralesweb.htm#Monocots" target="home">monocots</a>. </p>

<p style="text-indent: 30px;">In the following discussion, the ANA grade is considered fixed, as is the monophyly of the magnoliids, monocots and eudicots. However, in Goremykin et al. (2005) the monocots did not always form a monophyletic group, a rather unlikely result that has been obtained in some studies where the nuclear 18S gene has been included (Troitsky et al. 1991: see Duvall et al. 2006 for references).</p>

<p style="text-indent: 30px;">The questions to be addressed are, what are the relationships between the magnoliids, monocots and eudicots, and where in particular are Chloranthales and <i>Ceratophyllum</i> to be placed (see Zeng et al. 2014: suppl. Fig 1 for various hypotheses of relationships)? These two sets of questions are linked, for example, Graham et al. (2005) found that the inclusion of <i>Ceratophyllum</i> and Chloranthaceae could destabilise relationships among other early branching clades, e.g., the position of the monocots became labile. Over the years, the positions of Ceratophyllaceae and Chloranthaceae have remained particularly uncertain, and Piperales have also tended to wander around the tree. Another issue is the role of aquatics in early angiosperm evolution. How diverse and/or abundant were the two aquatic clades, Nymphaeales and Ceratophyllaceae and their associates, in the earlier Cretaceous? In some reconstructions of early angiosperm evolution, aquatic plants play a prominent role (see also <a href="amborellalesweb2.htm#EarlyAng">elsewhere</a>).</p>

<p style="text-indent: 30px;">As data sets increase in size and sampling of some clades is necessarily constrained - perhaps more Chloranthaceae could be included, but addition of more Ceratophyllaceae is unlikely to affect the issue - the amount of data and the analytical techniques used will become increasingly important (e.g. Jarvis et al. 2014). Thus different seed plant topologies were obtained from analyses using single genes or the same number of sites chosen from twelve separate loci (Burleigh &amp; Mathews 2007a); maximum likelihood and maximum parsimony analyses might show systematic error (Burleigh &amp; Mathews 2007b). Below only more frequently found relationships are discussed, but in more comprehensive analyses such as  Moore et al. (2010) and Sun et al. (2014) other "minority" relationships can usually be found. Shen et al. (2017) evaluate the support for two hypotheses of the position of Chloranthaceae - [Magnoliids [Eudicots + Chloranthales]] vs [[Magnoliids + Chloranthaceae] Eudcots].</p>

<p style="text-indent: 30px;"><u>Chloranthaceae and Ceratophyllaceae.</u>. A variety of relationships between the two have been suggested. </p> 

<p style="text-indent: 30px;">1. The two taxa are not immediately related. Hilu et al. (2003: a <i>mat</i>K analysis alone), suggested that Ceratophyllaceae were sister to eudicots (see also D. Soltis et al. 2000; Borsch et al. 2005: three rapidly evolving genes, 62% jacknife, 96% posterior probabilities; M&uuml;ller et al. 2006: 96% posterior probability; some analyses in Saarela et al. 2007; Soltis et al. 2007a: support weak; Qiu &amp; Estabrook 2008; most analyses in Drew et al. 2014). The clade  {Ceratophyllaceae + eudicots] was recovered by Givnish et al. (2016b), although Chloranthaceae were not included and <i>Piper</i> sometimes attached itself to <i>Ceratophyllum</i>. A [Chloranthaceae + magnoliid] clade has quite commonly been recovered, as by Jansen et al. (2006b) and Hansen et al. (2007). Thus Ruhfel et al. (2014: whole chloroplast genomes, support not strong) found that Chloranthaceae tended to be sister to Magnoliales and Ceratophyllaceae to the eudicots; see also Drew et al. (2014: most analyses) and Foster et al. (2016: support not strong, 2017). Similarly, <i>Ceratophyllum</i>, with quite a long branch, was sister to eudicots (in some reconstructions, <i>Piper</i>, with a very long branch, was also involved), and <i>Chloranthus</i> was  sister to the magnoliids (Jansen et al. 2007: moderate to strong support), a position also found by Saarela et al. (2007), many analyses in Moore et al. (2007, 2010), Smith et al. (2010) and Magall&oacute;n et al. (2015).</p> 

<p style="text-indent: 30px;">Rather unusually, Z.-D. Chen et al. (2016) found Chloranthaceae to be sister to the monocots and more conventionally Ceratophyllaceae to be sister to the eudicots. The relationships [eudicots [Ceratophyllaceae + Piperaceae]] were found by Xue et al. (2012: ML analysis of whole chloroplast genomes), although relationhips around here werer not the focus of the study, nor were they in a PHYC analysis in Hertweck et al. (2015) in which the relationships [Chloranthaceae [[<i>Ceratophyllum</i> + some of the Magnoliales included] [[<i>Acorus</i> + Eudicots] other monocots]]] were obtained.</p> 

<p style="text-indent: 30px;"> Drew et al. (2014) also found topologies like [Chloranthaceae [monocots [Ceratophyllaceae + Piperaceae]]], while there was fairly good support for a clade [monocots + Ceratophyllaceae] in a compartmentalised 6-gene analysis (Zanis et al. 2003; see also Qiu et al. 1999; some analyses in Saarela et al. 2007). Qiu et al. (2005; see also L&ouml;hne &amp; Borsch 2005) found initial rather strong bootstrap support for a [monocot + Ceratophyllaceae] clade in a 9-gene analysis being vitiated by the failure to obtain much support in any of the subanalyses and by details of the topology obtained in the 9-gene analysis itself (e.g. <i>Acorus</i> sister to the Alismatales included) that are rather improbable. Goloboff et al. (2009)  found a [Chloranthaceae + monocot] clade.  Moore et al. (2010) i.a. found <i>Ceratophyllum</i>, or [<i>Ceratophyllum</i> + <i>Piper</i>] to be sister to monocots and Chloranthaceae sister to a clade including magnoliids, monocots, and eudicots; the latter pair of relationships with moderate jacknife support were found by Davis et al. (2013) in their comprehensive chloroplast analysis focussing on monocots. Indeed, a [Ceratophyllaceae + Piperaceae] clade is not uncommonly found, as by Petersen et al. (2015), and there Chloranthaceae were sister to all angiosperms apart from the ANITA grade (see also P. S. Soltis et al. 2015) - but neither position had much support. Goremykin et al. (2009b: Chloranthaceae not included) found a [<i>Ceratophyllum</i> + magnoliid] clade after removing 2,500 highly variable positions from an analysis of chloroplast genome data; with the removal of 1,000 positions <i>Ceratophyllum</i> was sister to a [monocot + eudicot] clade. Some analyses have suggested a sister taxon relationship between Chloranthaceae and eudicots (Borsch et al. 2003; P. S. Soltis et al. 2015), and this relationship was favoured in several concatenation-based transcriptome analyses by Wickett et al. (2014; no Ceratophyllaceae). </p>

<p style="text-indent: 30px;">Other studies also show no clear pattern (e.g.  Jansen et al. 2006a; Qiu et al. 2006b). One alignment of 18S/26S nuclear ribosomal data suggested a clade [Ceratophyllaceae + Tofieldiaceae] embedded within the monocots, but thankfully with little support (Maia et al. 2014), while Du et al. (2016) considered Nymphaeales, Acorales and Alismatales, and Ceratophyllales to be extant members of an early radiation of aquatic angiosperms. Finally, Morton (2011) found weak support for Ceratophyllaceae as sister to all other angiosperms, while in the massive parsimony analysis of Goloboff et al. (2009) the relationships [<i>Ceratophyllum</i> [[<i>Amborella</i> + Nymphaeales] all other angiosperms]] were recovered.</p>


<p style="text-indent: 30px;">2. Sister taxa. A [Chloranthales + Ceratophyllales] clade is quite often obtained. Duvall et al. (2006) and Qiu et al. (2010: chloroplast data, support weak) found a [Chloranthales + Ceratophyllales] clade, as did N. Zhang et al. (2012); see also Huang et al. (2010), weak support using the <i>ycf</i>2 gene, Moore et al. (2011), very weak support, and  Kvacek et al. (2016: morphological analyses, inc. fossils). In the comprehensive analyses of Sun et al. (2014) this clade was also frequently recovered, although in a variety of positions, while the analyses in Mathews (2006a) preferred an unresolved position somewhere above the Austrobaileyales.  Indeed, if there is a there is a [Chloranthales + Ceratophyllales] clade, the question is, where is it to be placed? Usually it is associated with magnoliids, monocots or core eudicots (so see below), but sometimes it is placed sister to all other angiosperms except the ANITA grade (e.g. Endress &amp; Doyle 2009; Doyle &amp; Endress 2010; Doyle &amp; Upchurch 2014).</p>

<p style="text-indent: 30px;">Members of the two groups have fascinating morphologies, florally both being very reduced, and analysis of morphological features may suggest a relationship between the two. Thus the "flower" of <i>Ceratophyllum</i> with 3 to many stamens is perhaps best treated as an inflorescence made up of flowers that consist of a single stamen and nothing more, and this has major effects on the scoring of several characters (e.g. Endress 1994d; Endress &amp; Doyle 2015). Analyses of morphological data gave a  [<i>Ceratophyllum</i> + Chloranthaceae] clade with quite strong support (Endress &amp; Doyle 2009). Kvacek et al. (2012) linked <i>Pseudoasterophyllites</i>, ca 97 m.y.o. (Cenomanian) and vegetatively somewhat like <i>Ceratophyllum</i>,  with <i>Tucanopollis</i>, an abundant palynomorph from Africa-South America over 125 m.y.o., and <i>Pseudoasterophyllites</i> tends to link Chloranthaceae and <i>Ceratophyllum</i> (Doyle et al. 2015), and perhaps also <i>Appomattoxia</i>, another fossil (Kvacek et al. 2016). Similarly, the recently-described <i>Montsechia</i> is either Ceratophyllales or to be placed in a [Chloranthaceae + <i>Ceratophyllum</i>] clade (Gomez et al. 2015) while the fossil chloranthalean <i>Canrightiopsis</i> goes with <i>Ceratophyllum</i> - and with the odd magnolialian plant - in a splits-graph analysis (Friis et al. 2015a). If the staminate flower of <i>Ceratophyllum</i> is interpreted as being simply a single stamen, then much of this reduction must have occurred in the common ancestor of Chloranthaceae and Ceratophyllaceae - which was terrestrial (Endress &amp; Doyle 2015). Zeng et al. (2014) optimised the distribution of a large number of characters on a tree with a clade [<i>Ceratophyllum</i> + Chloranthaceae] (see also below).</p>  





<p style="text-indent: 30px;"><u>Relationships Between the Three Major Groups, Monocots, Magnoliids, and Eudicots.</u> </p>

<p style="text-indent: 30px;">1. [Monocots [magnoliids + eudicots]].  Zanis et al. (2002) analysed 11 genes from all
genomes and found some support for the magnoliids (strong support for this clade) as being sister to eudicots, with Chloranthaceae, and [monocots + Ceratophyllaceae]
occurring as successively more basal branches (see also Graham &amp; Olmstead 2000; Borsch et al. 2003). Support for some of these nodes depended on
the method of analysis (see also Borsch et al. 2000; Graham &amp; Olmstead 2000; Hilu et al. 2001; Whitlock et al. 2001) or the particular gene studied (see Duvall &amp; Bricker 2002: nuclear 18s; Hilu et al. 2003: <i>mat</i>K; Duvall et al. 2006: nuclear 18S in particular causes problems). Soltis et al. (2007a) found very weak support for  monocots as sister to [magnoliids, Chloranthaceae, eudicots, etc.].  Lee et al. (2011: Chloranthales and Ceratophyllales not included) found a clade [monocots [magnoliid + eudicot], and with strong support (see also Duarte et al. 2010), and they initially looked at almost 23,000 sets of orthologues from nuclear genomes of 101 genera of land plants (most taxa represented by ESTs, genes included if represented in as few as 4 taxa). Such relationships were strongly supported in the 59 low-copy nuclear gene analysis of Zeng et al. (2014: see also some analyses in Moore et al. 2011), and there sampling is not obviously a problem, <i>Ceratophyllum</i>, <i>Sarcandra</i> and <i>Chloranthus</i> being included. The [<i>Ceratophyllum</i> + Chloranthaceae] clade also recovered by these two groups was placed sister to eudicots (c.f. also Zeng et al. 2014: Table 2, p. 7 etc., conflict between single-copy and  IR genes). In the nuclear and mitochondrial analyses by M. Sun et al. (2014)  [Chloranthales + Ceratophyllales] were  sister to monocots [check]. In an early study the relationships [Monocots [<i>Ceratophyllum</i> [[magnoliids + <i>Chloranthus</i>] eudicots]]] were obtained (Barkman et al. 2000b), although support was poor, while Zanis et al. (2002) and Qiu et al. (2005: other topologies also recovered) found <i>Ceratophyllum</i> to be sister to monoocots and <i>Chloranthus</i> sister to the [eudicot + magnoliid] clade. Wickett et al. (2014) found that [Magnoliids + Chloranthaceae] were sister to eudicots in coalescent transcriptome analyses, other topologies (2 and 3 below) being rejected. </p> 

 <p style="text-indent: 30px;">2. [Eudicots [monocots + magnoliids]].  This topology was recovered by Hertweck et al. (2015: eight genes), with a [<i>Ceratophyllum</i> + Chloranthaceae] clade sister to the monocots. Eudicots were sister to an unresolved clade including monocots, magnoliids, and Chloranthaceae (D.
Soltis et al. 2000; P. Soltis et al. 2000; Wu et al. 2007; Qiu &amp; Estabrook 2008: compatibility analysis; Doyle &amp; Endress 2010). Hilu et al. (2003) in a <i>mat</i>K analysis, suggested that magnoliids were sister to monocots, although the support was weak in parsimony analyses if with 100% posterior probabilities in Bayesian analyses, furthermore, in the latter case only, [Chloranthacaeae + monocots] were sister to magnoliids, although the probabilities there were low. Davis et al. (2004) found  magnoliids to be sister to monocots and that Chloranthaceae and Ceratophyllaceae were successively sister to a clade including the few eudicots in the analysis, but support was not exactly overwhelming (the best supported topology in Duvall et al. 2006 is somewhat similar). M&uuml;ller et al. (2006) found very poorly supported relationships between Chloranthaceae and monocots, which together linked with the magnoliids.  Whitlock et al. (2002) recovered largely similar groupings, but support was only moderate; the exact position of Chloranthaceae remained unclear. Duvall et al. (2006: four genes, three compartments) also preferred a relationship between magnoliids and monocots, as did N. Zhang et al. (2012), Doyle and Upchurch (2014) and Doyle and Endress (2014).  Wickett et al. (2014) rejected a clade [monocots [Chloranthaceae + magnoliids]] in their transcriptome study.</p>

<p style="text-indent: 30px;">3. [Magnoliids [monocots + eudicots]]. A [monocot + eudicot] grouping has quite often been recovered, as in Jansen et al. (2006b: 37 whole chloroplast genomes, see also Zhengqiu et al. 2006), Cai et al. (2006: 35 whole chloroplast genomes, no Chloranthaceae, Ceratophyllaceae, or Austrobaileyales), Duvall et al. (2006: four genes, three compartments, nuclear <i>PHY</i>C gene, 18S being excluded), Mathews (2006a: three <i>PHY</i> genes, 105 taxa), Hansen et al. (2007: 61 protein-coding chloroplast genes), Mardanov et al. (2008: ?sampling), Goloboff et al. (2009), Xue et al. 2012: whole chloroplast genomes), Z.-D. Chen et al. (2016), Givnish et al. (2016b: 75 plastid genes: stem monocots (146-)142(-136) m.y.o.) and Foster et al. (2017: support weak). Graham et al. (2005) found a rather weakly supported (73% bootstrap) [monocot + eudicot] grouping, and this was yet more weakly supported when Chloranthaceae and Ceratophyllaceae were included. When sequences of complete chloroplast genomes are analysed, an association between monocots and eudicots may be more strongly suggested. Thus Jansen et al. (2007) found good support for a [monocot + eudicot] grouping, a number of alternative topologies being excluded, but support in Moore et al. (2007) and  Foster et al. (2016) was somewhat less strong; see also Magall&oacute;n et al. (2015). However, the possibility of a [monocot + eudicot] grouping was rejected by Wickett et al. (2014: transcriptome analyses).  </p>

<p style="text-indent: 30px;">Evolution of some floral developmental genes, e.g. in the C and D lineages, are also consistent with a [monocot + eudicot] clade (Kramer et al. 2004), as is the distribution of taxa in which DEF-like proteins cannot form heterodimers (Melzer et al. 2014). The positional relationships between members of the androecium and the perianth, the stamens being individually opposite perianth members, and trimery of some floral whorls occur in a number of taxa, and their distribution is broadly consistent with all three sets of relationships.  Variation in how calcium oxalate is synthesized may also be of phylogenetic interest, although sampling is very poor; neither members of the ANITA grade nor magnoliids have been examined. Vacuolar crystal formation associated with membranes and paracrystalline bodies with widely spaced subunits are found in eudicots (crystals seem also to be formed in other ways here), while in monocots there are no membrane complexes and the paracrystalline bodies have closely spaced subunits (Horner &amp; Wagner 1995; Evert 2006). Some patterns of RNA editing may be phylogenetically informative at this level (Logacheva et al. 2008). </p>

<p style="text-indent: 30px;">4. Other Relationships. Analysis of morphological data in particular has quite often grouped more or less herbaceous clades in magnoliids and the ANITA grade, i.e. Piperales, Chloranthaceae, and Nymphaeales, with monocots; this is the pal(a)eoherb hypothesis (see e.g. Donoghue &amp; Doyle 1989; Taylor &amp; Hickey 1992; Endress 2000). Analysis of combined morphological and molecular data suggested that Piperales weresister to monocots (Doyle &amp; Endress 2000: support weak), Tamura et al. (2011: 6 plastid, 7 mitochondrial, and 1 nuclear genes sequenced) found some support for this clade, as did Barrett and Davis (2011: or [Piperales + Ceratophyllales] the sister group). However, although quite popular in the latter years of the last century, the palaeoherb support comes from homoplasies associated with the adoption of the herbaceous habit. Indeed, Piperales are now firmly (usually!) embedded in the magnoliid clade, suggestions like that of Parkinson et al. (1999) of a [Piperales + eudicot] and by Ruhfel et al. (2014: amino acid analyses) of a [Piperales + Ceratophyllaceae] clade being exceptions, while Nymphaeales are very near the base of the whole angiosperm clade.</p>



<p style="text-indent: 30px;"><u>To Conclude.</u> There may be be differing signals in nuclear (relationships - [monocots [magnoliids + eudicots]]) and chloroplast ([magnoliids + [monocots + eudicots]]) genes (Xi et al. 2014: <i>Ceratophyllum</i> and Chloranthaceae not included). Sun et al. (2014) retrieved the latter set of relationships, but both nuclear and mitochondrial trees recovered a [eudicots [monocots + magnoliids]] topology. Wickett et al. (2014) in particular noted that analyses of chloroplast genes did not give consistent relationships. A pentatomy seems the best depiction of our current understanding of relationships.  <i>Ceratophyllum</i> and Chloranthales must be included in future analyses of the relationships between the monocots, magnoliids and eudicots; in their absence, the analyses must be considered incomplete. Since a clade [Chloranthaceae + Ceratophyllaceae] is quite commonly recovered, characters of that clade, several of which may be apomorphies, are suggested below (in part from Zeng et al. 2014), but to which major clade it should be attached is unclear. I have also given characters for a clade [Chloranthales + magnoliids] above since this relationships is quite comonly recovered.</p> 

<p><u>[<i>Ceratophyllum</i> + Chloranthaceae]</u>: leaves opposite, secondary veins pinnate; flowers dense, sessile, small [<5 mm across], monosymmetric by reduction; P 0; A 1, sporangia &plusmn; embedded in connective; <u>G 1</u>, ascidiate; ovule 1/carpel, apical, straight.</p>

<p style="text-indent: 30px;"><b>Age.</b> The age for this node is estimated to be (174-)170, 126(-71) m.y.a. (Zeng et al. 2014).</p>

<p>[magnoliids [[<i>Ceratophyllum</i> + Chloranthaceae] + eudicots]]: <b>Age.</b> The age for this clade was estimated to be (192-)187,148(-128) m.y. (Zeng et al. 2014).</p>

<p>[[<i>Ceratophyllum</i> + Chloranthaceae] + eudicots]: <b>Age.</b> This clade is about (187-)182, 141(-132) m.y.o. (Zeng et al. 2014).</p>

<p><a name="ChlMag">[CHLORANTHALES [[MAGNOLIALES + LAURALES] [CANELLALES + PIPERALES]]]</a>: <b class="apo">sesquiterpenes +</b>; (microsporogenesis also simultaneous); seed endotestal.</p>

<p style="text-indent: 30px;"><b>Age.</b> Moore et al. (2010) estimated an age of  (141-)136(-129) m.y. for this node and Xue et al. (2012) an age of ca 143.2 or 141.3 m. years. Clarke et al. (2011) a somewhat older age of (176-)149(-128) m.y., Soltis et al. (2008) suggested (168-)131(-126) m.y., and Magall&oacute;n et al. (2013) an age of around 149.1 m.y. and Naumann et al. (2013) an age of around 142.5 m.y.; see also Magall&oacute;n (2009) for other dates around 140-150 m.y. ago, about 134.6 m.y. was estimated by Magall&oacute;n et al. (2015), ca 170 m.y. by Foster et al. (2016, q.v. for details) and ca 198.9 to 192 m.y. by Tank et al. (2015: Table S1, S2). The estimate in Z. Wu et al. (2014), at ca 210 m.y.a., is definitely the oldest.</p>


<p style="text-indent: 30px;">Doyle and Endress (2010) thought that the <i>Pennipollis</i> plant, from ca 120-115 m.y.a. in Portugal and originally linked to the monocots (Petersen et al. 2000b), was sister to Chloranthaceae, a family they considered to be sister to all angiosperms apart from the ANA grade.</p>

<p style="text-indent: 30px;"><b>Evolution:</b> <u>Divergence &amp; Distribution.</u> Soltis et al. (2008) offer ages for a number of branching points within this clade based on the topology [monocots [Chloranthaceae, magnoliids [Ceratophyllaceae + eudicots]]].</p>

<p style="text-indent: 30px;">The character, "endotesta palisade, crystaliferous", could perhaps be placed at this node.</p>

<p style="text-indent: 30px;"><b>Chemistry, Morphology, etc.</b> Microsporogenesis is variable throughout this clade (Furness &amp; Rudall 2004).</p>

<p><font face="verdana, georgia, arial, serif" size="3"><b><a name="Chloranthales">CHLORANTHALES</a></b></font> Martius&nbsp;&nbsp;<a href="treeapweb2map.html" target="home">Main Tree</a>.</p>

<p>Branching from the current flush; neolignans ?+; compression wood + [<i>Sarcandra]; </i>nodes often
swollen; <b class="apo">leaves opposite, joined by sheath</b>, lamina margins toothed, teeth with lateral vein and others [hydathodal]; stipules +; <b class="apo">flowers very small</b> [<4 mm across], <b class="apo">monosymmetric by reduction</b>, parts whorled; <b class="apo">P 0</b>, <b class="apo">A 1</b>, abaxial; <b class="apo">G 1</b>, ascidiate, postgenital fusion by secretion; compitum necessarily 0; ovule 1/carpel, apical, pendulous, <b class="apo">straight</b>; antipodal cells proliferating; fruit fleshy; <b class="apo">endotesta palisade, lignified, crystalliferous</b>.  - 1 family, 4 genera, 75 species.</p>

<font size="1pt"><i><script language="javascript" type="text/javascript">
	<!--Hide script from old browsers
	
	//Date last modified stamp using javascript
	document.write("Page last updated: " + document.lastModified)

	//end hiding script-->
	</script></i></font>

<p style="text-indent: 30px;"><b>Note:</b>  <b>Boldface</b> denotes possible apomorphies, (....) denotes a feature common in the clade, exact status uncertain, [....] includes explanatory material. Note that the particular node to which many characters, particularly the more cryptic ones, should be assigned
is unclear. This is partly because homoplasy is very common, in addition, basic information for all too many characters is very incomplete, frequently coming from taxa well embedded in the clade of interest and so making the position of any putative apomorphy uncertain. Then there are the not-so-trivial issues of how character states are delimited and ancestral states are reconstructed (see <a href="introductionnew.htm#Warning" target="home">above</a>).</p>	
	<p><b>Includes</b> <a href="chloranthalesweb.htm#Chloranthaceae">Chloranthaceae</a>.</p>
	
	<p>Synonymy: <b>Chloranthineae</b> Thorne &amp; Reveal - <b>Chloranthanae</b> Doweld - <b>Chloranthidae</b> C. Y. Wu</p>

<p><a name="Chloranthaceae"><b>CHLORANTHACEAE</b></a> Sims, <i>nom. cons.</i> &nbsp;&nbsp;<a href="chloranthalesweb.htm#Chloranthales">Back to Chloranthales</a></p>

<img src="chloranthaceaemap.gif" width="378" height="186" border="1" align="right" alt="Chloranthaceae"><p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:0in; margin-left:.25in;margin-bottom:.0001pt;line-height:200%'>Evergreen; (vessels 0); primary stem with vascular cylinder; compression wood + [<i>Sarcandra</i>]; rays 6-10-seriate; nodes 1:1, 1:2, or &plusmn; 3:3, 2 traces from the central or all gaps, (+ split
laterals); 
(sclereids - <i>Hedyosmum</i>); cuticle wax crystalloids 0; stomata variable [laterocytic, etc.];
branching from current flush; lamina vernation conduplicate [<i>Chloranthus</i>], teeth
with clear persistent swollen cap into which proceed higher order veins as
well as secondaries or tertiaries; <b class="apo">stipules small, paired, interpetiolar,
usually on rim of sheath</b>; (plants dioecious); <b class="apo">inflorescence spicate</b> (branched), <b class="apo">flowers sessile</b>; staminate flowers: A &plusmn; latrorse, lobed, or connective produced or not, (glandular); (pollen apertures star-shaped polychotomosulcate, pantocolpate, pantoporate); pistillode 0; carpellate flowers: (P +, uniseriate, &plusmn; connate); staminode 0; (ovary inferior), stigma &plusmn; expanded or not, dry ?or
wet; ovules with outer integument 4-8 (2 - <i>Ascarina</i>) cells across, inner integument (3-)7-10 cells across, (micropyle
bistomal), parietal tissue 6-8 cells across, nucellar cap +/0; fruit baccate or drupaceous, (bracts accrescent and succulent), (P persistent); coat &plusmn; tanniniferous, (mesotesta
lignified - <i>Chloranthus</i>), endotesta &plusmn; palisade, containing crystals, tegmen &plusmn; crushed, (exo- and mesotegmen fibrous),
endotegmen initially subpalisade; endosperm cellular, starchy, grains clustered); n = 8, 14, 15, chromosomes 1-4(-10: <i>Hedyosmum</i>) &micro;m long. </p>

<p>4<a href="javascript:showImage('http://data.kew.org/cgi-bin/vpfg1992/genlist.pl?CHLORANTHACEAE',600,500)">[list]</a>/75: <i>Hedyosmum</i> (45). Tropics and subtropics, not Africa (Madagascar - <i>Ascarina</i> only) (map: from Verdcourt 1986; Todzia 1988; Leitman 2014). <a href="javascript:showImage('http://mobot.mobot.org/cgi-bin/search_vast?w3till=MOA-02054_001.jpg',600,500)">[Photo - Leaf</a>, <a href="javascript:showImage('http://mobot.mobot.org/cgi-bin/search_vast?w3till=MOA-02053_001.jpg',600,500)">Flower.]</a></p>

<p style="text-indent: 30px;"><b>Age.</b> Despite the age of the clade (see below), some estimates are that crown group diversification is quite recent, being mostly within the last 60-29 m.y. (Zhang &amp; Renner 2003b; Soltis et al. 2008). However, Magall&oacute;n and Castillo (2009) estimated the crown group age at  ca 153.6 or 125 m.y., Wikstr&ouml;m et al. (2001) at 131-121 m.y., Bell et al. (2010) at around 121 or around 98 m.y.a., depending on the analysis,  Antonelli and Sanmart&iacute;n (2011: fossil-based) suggested ages of (112-)111(-110) m.y. or thereabouts, while Zhang et al. (2011) provided another series of age estimates, some of which are dramatically older than the others depending on the calibration and the analytical methods used.</p>

<p style="text-indent: 30px;">Fossils assignable to Chloranthaceae are already common, diverse, and world-wide in distribution in the early angiosperm fossil record (Friis et al. 2011, 2015a; Doyle &amp; Upchurch 2014 for summaries). Distinctive fossil pollen grains, <i>Asteropollis</i>, are first known from the Barremian-Aptian of the early Cretaceous, some 125 m.y. before present (Friis et al. 1997; Doyle 1999; Eklund 1999, but c.f. Clarke et al. 2011, questions over dating); these grains have been identified as belonging to <i>Hedyosmum</i> (see also Crepet &amp; Nixon 1996; Eklund et al. 2003; Friis et al. 2005; Mart&iacute;nez et al. 2013). Doyle and Endress (2007) and Clarke et al. (2011) discuss other palynomorphs that have been associated with Chloranthaceae including <i>Clavatipollenites</i>; some fossil androecia assigned to the family have spiraperturate pollen that is <i>in situ</i> (Crane et al.  1989)! Similarly, <i>Clavatipollenites</i>, widespread in the fossil record in the Cretaceous, is like that of <i>Ascarina</i> but has also been found on fruits of <i>Couperites</i> that may not even belong to the family (Doyle et al. 2003).  </p>


<ul><li>Chloranthaceae are aromatic plants that may be
recognised by their opposite, serrate leaves with more or less sheathing
stipules and often strongly swollen nodes that may collapse on drying. The flowers are very small and are often
borne on branched inflorescences; the fruits are fleshy. The
wood is soft.</ul>



<p style="text-indent: 30px;"><b>Evolution:</b> <u>Divergence &amp; Distribution.</u>  With older estimates of the family age, all genera had separated by ca 90 m.y.a. (e.g. Wikstr&ouml;m et al. 2000; Antonelli &amp; Sanmart&iacute;n 2011). Diversification within <i>Hedyosmum</i> could be dated to (57.1-)43.3(-30.1) or (43-)35.6-(25.9) m.y.a. depending on the method used, and was in part associated with the Andean uplift, although the family as a whole showed a pattern of gradual extinction over time (Antonelli &amp; Sanmart&iacute;n 2011). 

<p style="text-indent: 30px;">For general morphological evolution, living and fossil Chloranthaceae integrated, see Eklund et al. (2004). Endress (2001) emphasized what he considered to be the plesiomorphic floral morphology of the family. However, there is no evidence that it is a member of the ANA grade, and several aspects of its floral morphology and development, including the loss of any perianth, are clearly highly derived derived (e.g. G.-S. Li et al. 2005). For the evolution of the chloranthaceous flower, integrating fossils into the scenario, see Doyle and Endress (2014): the flowers of <i>Chloranthus</i> and <i>Sarcandra</i> are secondarily perfect/bisexual (see also Doyle &amp; Endress 2011), or imperfect flowers have evolved twice, or perfect flowers are really pseudanthia (Endress &amp; Doyle 2015).... In the careful analysis of Friis et al. (2015a), <i>Canrightiopsis</i>, with a single carpel and three stamens borne on one side of semi-inferior gynoecium, shows similarities with <i>Canrightia</i>. The latter has a hypanthium, semi-inferior ovary, 2-5 carpels each with single ovules, the ovules perhaps with an endothelium (the endotegmic cells are radially elongated), a crystal-bearing endotesta, and its stamens are radially arranged; at 126-110 m.y.o., it attaches to the stem of Chloranthaceae in phylogenetic analyses (Friis &amp; Pedersen 2011). The seeds of <i>Canrightiopsis</i> are very small, and its embryo is about one quarter the size of that of <i>Sarcandra</i>, with which it has been compared (Friis et al. 2015b).</p>

<p style="text-indent: 30px;"><b>Evolution:</b> <u>Pollination Biology &amp; Seed Dispersal.</u> Nectar is reported from <i>Sarcandra</i> at least, and pollination  may be by small insects (and in <i>Chloranthus</i>), while wind pollination occurs in <i>Ascarina</i> and <i>Hedyosmum</i> (Erbar 2014; Gottsberger 2016 and references).</p>

<p style="text-indent: 30px;"><b>Chemistry, Morphology, etc.</b> Although benzylisoquinoline alkaloids apparently have not been detected in Chloranthaceae, (<i>S</i>)norcolaurine synthase activity is high, suggesting that they may be found here (Liscombe et al. 2005). <i>Sarcandra glabra</i> lacks vessels and has - unuusually for angiosperms - compression wood, and this may be associated with the fact that the ration of syringyl to guaiacyl units in the lignin may be low (Aiso et al. 2014, 2016). Roots - presumably those of the seedlings and young plants - seem not have any secondary thickening (Blanc 1986)?</b>

<p style="text-indent: 30px;">There is some discussion of inflorescence morphology in Doria et al. (2012). The perianth of <i>Hedyosmum</i> has unique, schizogenous apertures/windows (Doria et al. 2012 and references; c.f. Endress &amp; Doyle 2015). Crane et al. (1989), Crepet and Nixon (1997), Eklund
et al. (1997), Eklund (1999), and others discuss the nature of the androecium in the family, a matter over which there has been considerable controversy.  In <i>Chloranthus</i>
it has been suggested that the androecium is lobed, with 2 or 4 dithecal
stamens(!!?), and that staminate flowers of <i>Hedyosmum</i> have hundreds of anthers, however, here as in most other angiosperms the stamens seem to be dithecal (Kong et al. 2002, and references). Doria et al. (2012) illustrate a superior gynoecium in <i>Hedyosmum</i>. The stylulus is filled with secretion. Endress and Igersheim
(1997) describe the stigma as being wet (c.f. Todzia 1988). The ovule in <i>Chloranthus</i> is not quite straight ("subatropous" - Yamada et al. 2001a). For variation in micropyle type, see Heo and
Tobe (1995). Johri et al. (1992) noted
that the endosperm stores oil, but there may also be starch. The cotyledonary nodes have split laterals (Bailey 1956).</p> 


<p style="text-indent: 30px;">For general information, see Swamy (1953a), Todzia
(1988, 1993) and Eklund (1999), for chemistry, see Hegnauer (1964, 1989), young plants, see Blanc (1986), general vegetative anatomy, see Metcalfe (1987), for wood anatomy, see Carlquist (1992a), for embryology, see Vijayaraghavan (1964), and for floral development, see Endress (1987b).</p> 

<p style="text-indent: 30px;"><b>Phylogeny.</b> Within Chloranthaceae, morphological analyses, including details of wood anatomy, suggested the genus pairs [<i>Ascarina</i> + <i>Hedyosmum</i>], mainly woody, plant monoecious to dioecious, and [<i>Chloranthus</i> + <i>Sarcandra</i>], herbaceous to semi-shrubby, flowers perfect. In early molecular work (Qiu et al. 1999), on the other hand, the relationships [<i>Hedyosmum</i> [<i>Ascarina</i> [<i>Chloranthus</i> + <i>Sarcandra</i>]]] were found. Although there was strong support, the sampling was rather minimal, but the same set of relationships were confirmed by Zhang and Renner (2003b) and Zhang et al. (2011), both with a much improved sampling. They have also been found in subsequent morphological analyses with constrained outgroups (Doyle et al. 2003; Eklund et al. 2004).</p>

<p style="text-indent: 30px;"><b>Previous Relationships.</b> Cronquist (1981) included Chloranthaceae in his Piperales while Takhtajan (1997) placed Chloranthales after Calycanthales at the end of his Magnoliidae.</p> 

<p>Synonymy: <b>Hedyosmaceae</b> Caruel</p>

<!-- Start of StatCounter Code -->
<script type="text/javascript" language="javascript">
var sc_project=931654;
var sc_invisible=1;
var sc_partition=7;
var sc_security="e82f7e9b";
</script>

<script type="text/javascript" language="javascript"
src="http://www.statcounter.com/counter/frames.js"></script><noscript><a
href="http://www.statcounter.com/" target="_blank"><img
src="http://c8.statcounter.com/counter.php?sc_project=931654&amp;java=0&amp;security=e82f7e9b&amp;invisible=1"
alt="free log" border="0"></a> </noscript>
<!-- End of StatCounter Code -->
</body>  

</html>

